README.md

```diff
@@ -78,6 +55,16 @@ start-stack
 
 Given successful deployment, the frontend should be accessible at: `http://localhost/`. If you encounter any problems try looking at the logs `logs api` / `logs frontend`.
 
+To deploy the stack to a remote server, create another `.env` file and modify it accordingly:
+```bash
+cp env-local.env.sample env-dev.env
+```
+
+Use the name of the file (`dev` in the previous example) as an argument to the `deploy-stack` command to pass the `.env` file to the Docker command and the Docker compose file:
```

Comment:
```suggestion
Use the tag of the file (`dev` in the previous example) as an argument to the `deploy-stack` command to pass the `env-dev.env` file to the Docker command and the Docker compose file:
```

Reply:
I think it might be good to mention that the  env variable `DOCKER_CONTEXT` in the file `env-dev.env` needs to be changed accordingly.

Reply:
Like all other variables, the use of `DOCKER_CONTEXT` is documented. I'm thinking we either document in the Readme all "important" variables, or none.

Reply:
I could not find the documentation of DOCKER_CONTEXT in MetAtlas, or do you mean the general documentation of DOCKER_CONTEXT for docker?

Reply:
Sorry, I meant that the variable is explained on the line prior it's use in `sample.env`.

Reply:
I would say a new member will encounter problem by just reading `To deploy the stack to a remote server, create another ENV file, e.g. env-dev.env, and modify it accordingly:` without assistant from others. 
However, I agree that we don't need over documenting, since only the developers of MetAtlas will deploy to the dev server.

Reply:
I think we have continued this discussion in the main thread, so I am closing it here.